.github/workflows/release.yaml

```diff
@@ -194,7 +194,7 @@ jobs:
       run: |
         set -e
         git checkout ${GITHUB_REF#refs/heads/}
-        v="$(go run ./api/version/generate bump-version)"
+        v="$(go run ./api/version/generate bump-minor)"
```

Comment:
hmm... what happens when we need to create a patch-release (e.g. `v0.17.1`)? Then this is executed on `releases/vX.Y` branch right? Or am I wrong? Wouldn't then the version be bumped to `v0.18.x` ?